codespaces.sample.env

```
@@ -0,0 +1,26 @@
+NODE_ENV="development"
+SERVER_HOST="0.0.0.0"
+SERVER_PORT="3000"
+POSTGRES_URL=""
```

```suggestion
POSTGRES_URL="postgresql://digitalmarketplace:digitalmarketplace@localhost:5432/digitalmarketplace"
```
In retrospect, I should have put this in here instead